src/Database/Database.php

```diff
@@ -90,9 +90,12 @@ private function readServerVariables(array $server)
 	public function connect()
 	{
 		if (!is_null($this->connection) && $this->connected()) {
-			return true;
+			return $this->connected;
```

Comment:
Both the function and the variable is named "connected"?

Reply:
yep, that's something I noticed too during this rework .. But I think we should change it for `2019.12` to not accidentally brick something..